File: 7KYU/Selective Fear of Numbers/solution.py
def am_I_afraid(day,num):
    return (day == "Monday" and num == 12) or (day == "Tuesday" and num > 95) or (day == "Wednesday" and num == 34) or (day == "Thursday" and num == 0) or (day == "Friday" and num % 2 == 0) or (day == "Saturday" and num == 56) or (day == "Sunday" and (num == 666 or num == -666))

File: 7KYU/Selective Fear of Numbers/solution.rb
def am_I_afraid(day,num)
    (day == "Monday" && num == 12) ||
    (day == "Tuesday" && num > 95) ||
    (day == "Wednesday" && num == 34) ||
    (day == "Thursday" && num == 0) ||
    (day == "Friday" && num % 2 == 0) ||
    (day == "Saturday" && num == 56) ||
    (day == "Sunday" && (num == 666 || num == -666))
  end